browser/perf/brave_perf_features_processor.cc

```diff
@@ -59,8 +59,9 @@ void MaybeEnableBraveFeatureForPerfTesting(Profile* profile) {
     return;
   }
 
-  // Ads
-  profile->GetPrefs()->SetBoolean(brave_ads::prefs::kEnabled, true);
+  // Notification Ads
```

Comment:
Maybe we should enable NTP ads here also?

Reply:
Ads are enabled by default when joining Brave Rewards. cc @zenparsing to confirm

Reply:
That is correct.